pipewire/qubes-pw-module.c

```diff
@@ -790,9 +790,9 @@ static void stream_param_changed(void *data, uint32_t id,
 doit:
     params[0] = spa_pod_builder_add_object(&b,
             SPA_TYPE_OBJECT_ParamBuffers, SPA_PARAM_Buffers,
-            SPA_PARAM_BUFFERS_buffers, SPA_POD_Int(1),
+            SPA_PARAM_BUFFERS_buffers, SPA_POD_CHOICE_RANGE_Int(8, 2, 64),
             SPA_PARAM_BUFFERS_blocks, SPA_POD_Int(1),
-            SPA_PARAM_BUFFERS_size, SPA_POD_Int(2048),
+            SPA_PARAM_BUFFERS_size, SPA_POD_Int(2048 * (int)impl->frame_size),
```

Comment:
Is there a reason to worry about this overflowing?  I presume not.

Reply:
Not really, frame_size is always 4 with the current format (and very small number with any other format, if we'd change).